modules/custom/az_migration/migrations/migrate_plus.migration.az_node_carousel.yml

```diff
@@ -0,0 +1,105 @@
+id: az_node_carousel
+label: AZ Carousel Item
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_node
+  node_type: uaqs_carousel_item
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_carousel_item
+
+process:
+  type: 
+    plugin: default_value
+    default_value: az_carousel_item
+  langcode:
+    plugin: static_map
+    bypass: true
+    source: language
+    map:
+      und: en
+
+  title:
+    plugin: sub_process
+    source: title_field
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_link:
+    plugin: sub_process
+    source: field_uaqs_link
+    process:
+      delta: delta
+      uri: 
+        -
+          plugin: str_replace
+          search: '[node:url]'
+          replace: ''
+          source: url
+      title: title
+
+  field_az_carousel_short_summary:
+    plugin: sub_process
+    source: field_uaqs_summary_short
+    process:
+      delta: delta
+      value: value
+      format:  
+        plugin: default_value
+        default_value: az_standard
+
+  field_az_media_image:
+    plugin: sub_process
+    source: field_uaqs_photo
+    process:
+      delta: delta
+      target_id: 
+      - 
+        plugin: migration_lookup
+        source: fid
+        migration: 
+          - az_media
+         
+  uid:
+    -
+      plugin: migration_lookup
+      migration: az_user
+      no_stub: true
+      source: node_uid
+    -
+      plugin: default_value
+      default_value: 0
+
+  status: status
+  created: created
+  changed: changed
+  promote: promote
+  sticky: sticky
+
+  path/pathauto: pathauto
+  path/alias: 
+    plugin: concat
+    source:
+      - constants/slash
+      - path
+
+dependencies:
+  enforced:
+    module:
+      - az_migration
+      - az_carousel
```

Comment:
Since the module dependencies defined in the migrations don't seem to do anything maybe we should just remove them?

Reply:
@joeparsons Last time I removed az_carousel dependency, we couldn't see the title for carousel in the content area, so I put that back in.

Reply:
> @joeparsons Last time I removed az_carousel dependency, we couldn't see the title for carousel in the content area, so I put that back in.

@smitahantzios I'm talking about the module dependencies defined in `modules/custom/az_migration/migrations/migrate_plus.migration.az_node_carousel.yml`.  It seemed like those didn't do anything while `az_carousel` was removed as a dependency from `az_migration.info.yml` (as you said) so I'm wondering if there is any value having them in the migration file at all?